### DISCUSSION TITLE
t5 model performace improvement on NVIDIA T4

### DISCUSSION THREAD

**Discussion by praveeny1986** [microsoft/fastseq#75] [Q&A]
Hi,

Is this 4.7x prediction improvement for T5 model valid for other GPUS as well?
We are planning to try on NVIDIA T4.

Please let me know.
Thanks

**Comment by JiushengChen** [comment]
Hi Praveen,

Our benchmark was done on V100 16GB. We didn’t test it on T4. It is expected the speed gain can vary on different GPUs. If you have some numbers on T4, we’d like to know. Thanks. Happy Holidays!
